### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

**Review by fridex — APPROVED** [review]
:100: :ship: :duck:

**Comment by bissenbay** [comment]
@fridex @harshad16 do I need to add `.aicoe-ci.yaml`?

**Comment by fridex** [comment]
> @fridex @harshad16 do I need to add `.aicoe-ci.yaml`?

Yes, please do so.

**Comment by bissenbay** [comment]
> > @fridex @harshad16 do I need to add `.aicoe-ci.yaml`?
> 
> Yes, please do so.

thanks @fridex , @harshad16 , https://github.com/thoth-station/management-api/pull/634/commits/12fc2862cd97c7224ca994e2fb33e51ef38cdb3c

**Review by fridex — COMMENTED** [review]
looks good to me! :+1: 

/assign @harshad16

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+check:
+  - thoth-build
+  - thoth-precommit
+  - thoth-pytest-py38
```
Comment (harshad16) [inline .aicoe-ci.yaml:4]:
```suggestion
```

File: .aicoe-ci.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build:
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
+  build-stratergy: Source
+  registry: quay.io
+  registry-org: thoth-station
+  registry-project: adviser
```
Comment (harshad16) [inline .aicoe-ci.yaml:10]:
```suggestion
  registry-project: management-api
```

File: .aicoe-ci.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  registry-secret: thoth-station-thoth-pusher-secret
+deploy:
+  project-org: "thoth-station"
+  project-name: "thoth-application"
+  image-name: "adviser"
+  overlay-contextpath: "adviser/overlays/test/imagestreamtag.yaml"
```
Comment (harshad16) [inline .aicoe-ci.yaml:16]:
```suggestion
  overlay-contextpath: "management-api/overlays/test/imagestreamtag.yaml"
```

File: .aicoe-ci.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
+  build-stratergy: Source
+  registry: quay.io
+  registry-org: thoth-station
+  registry-project: adviser
+  registry-secret: thoth-station-thoth-pusher-secret
+deploy:
+  project-org: "thoth-station"
+  project-name: "thoth-application"
+  image-name: "adviser"
```
Comment (harshad16) [inline .aicoe-ci.yaml:15]:
```suggestion
  image-name: "management-api"
```

**Review by harshad16 — COMMENTED** [review]
thanks for all the changes,
some minor suggestions.

**Comment by bissenbay** [comment]
@harshad16 I just committed changes suggested by you

**Pull request merged**